### ISSUE TITLE
[dataio] documentation for I3Writer parameter DropOrphanStreams (Trac #1886)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#1878]
[dataio] documentation for I3Writer parameter DropOrphanStreams

Migrated from https://code.icecube.wisc.edu/ticket/1886
```json
{
    "status": "closed", 
    "changetime": "2019-02-13T14:13:57", 
    "description": "[dataio] documentation for I3Writer parameter DropOrphanStreams", 
    "reporter": "david.schultz", 
    "cc": "", 
    "resolution": "fixed", 
    "_ts": "1550067237750774", 
    "component": "combo core", 
    "summary": "[dataio] documentation for I3Writer parameter DropOrphanStreams", 
    "priority": "normal", 
    "keywords": "", 
    "time": "2016-10-06T18:21:24", 
    "milestone": "", 
    "owner": "david.schultz", 
    "type": "rumor / allegation"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20161017T17:21:24: 
* olivas changed owner from "" to "olivas"
* olivas changed component from "icerec" to "icecube offline"
* olivas changed status from "new" to "accepted"

**Comment by icecube-trac** [comment]
Trac update at 20170330T19:02:15: 
* olivas commented:

> I think we fixed this at some point.


* olivas changed owner from "olivas" to "david.schultz"
* olivas changed status from "accepted" to "assigned"

**Comment by icecube-trac** [comment]
Trac update at 20170403T19:49:17: 
* david.schultz changed milestone from "October 2016" to "June 2017"
* david.schultz commented:

> I still can't find the docs. Even if it's there, if I can't find it no one else will.

**Comment by icecube-trac** [comment]
Trac update at 20170522T21:46:22: 
* david.schultz commented:

> In changeset 2897:
> 
> ```text
> #!CommitTicketReference repository="IceTray" revision="2897"
> documentation overhaul. add more information and clean up some of the older stuff to embrace new syntax. closes #1878
> ```


* david.schultz changed status from "assigned" to "closed"
* david.schultz changed resolution from "" to "fixed"

**Comment by icecube-trac** [comment]
Trac update at 20190111T21:55:04: nwhitehorn commented:

> In changeset 169218:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="169218"
> documentation overhaul. add more information and clean up some of the older stuff to embrace new syntax. closes #1878
> ```

**Comment by icecube-trac** [comment]
Trac update at 20190213T14:13:57: 
* olivas changed milestone from "June 2017" to ""
* olivas commented:

> Milestone June 2017 deleted
